gluu-server/opt/gluu/jetty/oxauth/custom/pages/WEB-INF/incl/components/alerts-en.xhtml

```diff
@@ -19,14 +19,21 @@
     </ui:fragment>
 
     <ui:fragment rendered="#{not empty facesContext.getMaximumSeverity() and facesContext.getMaximumSeverity() == FacesMessage.SEVERITY_ERROR}">
-        <div class="row mrgn-lft-0">
+        <div class="row">
             <section class="col-xs-11 mrgn-bttm-md problem-summary">
                 <h2>There was a problem</h2>
                 <p>Errors were found on this page:</p>
                 <ui:repeat value="#{facesContext.messageList}" var="message" varStatus="status">
                     <ui:fragment rendered="#{message.severity == FacesMessage.SEVERITY_ERROR and !message.summary.startsWith('An unexpected')}" >
                         <div class="problem-summary-content">
-                            #{status.index + 1}. #{message.summary}
+                            <ui:fragment rendered="#{not empty alertlink}">
+                                <a href="#{'#'.concat(alertlink)}" class="visited-link-styling">
```

Comment:
This is a bit of a hack. It only works because we never have more than one text input field on any of our pages. So the page just sets a view parameter ("alertlink") with the id of that field and the include above picks it up.

This was all unfortunately necessary because the WET form validation plugin does not correctly pick up server-side errors unless there is at least one client-side error. Once we have a front-end developer they should be able to clean this up.